cmd/mobile-security-service/main.go

```diff
@@ -52,3 +55,14 @@ func initLogger(level, format string) {
 		log.Fatalf("log format %v is not allowed. Must be one of [text, json]", format)
 	}
 }
+
+// Invoke handlers, services and repositories here
+func initHandlers(e *echo.Echo) {
+	// App handler setup
+	appsRepository := apps.NewRepository()
+	appsService := apps.NewService(appsRepository)
+	appsHandler := apps.NewAppsHandler(e, appsService)
+	
+	// Define /app routes
+	e.GET("/apps", appsHandler.GetApps)
```

Comment:
I have defined the routes here as I needed to make a small change to the handler to enable @damienomurchu\'s router to work correctly.